### PR TITLE
Update formatting for PSR-2

### PR DESCRIPTION
**Pull request by ntzm**
PSR-2 is now the standard for Laravel, so this should also follow suit.

I have also condensed a double `if` statement to just one line.

### CODE REVIEW

File: src/JacobBennett/Pjax/PjaxMiddleware.php [hunk 1]
```diff
@@ -1,55 +1,51 @@
...
+    public function handle($request, Closure $next)
+    {
         /** @var $response Response */
         $response = $next($request);
 
-        // Only handle non-redirections
-        if (!$response->isRedirection()) {
-            // Must be a pjax-request
-            if ($request->pjax()) {
-                $crawler = new Crawler($response->getContent());
```
Comment (JacobBennett) [inline src/JacobBennett/Pjax/PjaxMiddleware.php:26]: Oops! Can't remove this line!

File: src/JacobBennett/Pjax/PjaxMiddleware.php [hunk 2]
```diff
@@ -1,55 +1,51 @@
...
-
-                // Filter to given container
-                $response_container = $crawler->filter($request->header('X-PJAX-CONTAINER'));
+        // Only handle non-redirections and must be a pjax-request
+        if (!$response->isRedirection() && $request->pjax()) {
 
-                // Container must exist
-                if ($response_container->count() != 0) {
+            // Filter to title (in order to update the browser title bar)
+            $response_title = $crawler->filter('head > title');
```
Comment (JacobBennett) [inline src/JacobBennett/Pjax/PjaxMiddleware.php:29]: Where are you getting your $crawler from then?

**Comment by ntzm** [comment]
You're right, should be fixed now.

**Comment by ntzm** [comment]
Oops didn't mean to close :angry:

**Comment by JacobBennett** [comment]
Looks good, Thanks @ntzm

**Pull request merged**